### PR TITLE
Rewrite the Backend.limitPreviewHeight to vanilla JS

### PR DESCRIPTION
**Pull request by aschempp**
The new combiner is intentional since the script is not used in any of the other backend templates.

### CODE REVIEW

File: core-bundle/src/Resources/contao/templates/backend/be_main.html5 [hunk 1]
```diff
@@ -39,6 +39,11 @@
       $objCombiner->add('system/themes/'.$this->theme.'/hover.min.js');
       echo $objCombiner->getCombinedFile();
     ?>"></script>
+    <script src="<?php
+      $objCombiner = new Contao\Combiner();
```
Comment (Toflar) [inline core-bundle/src/Resources/contao/templates/backend/be_main.html5]: Why do you need the combiner? What does it do?

File: core-bundle/src/Resources/contao/templates/backend/be_main.html5 [hunk 2]
```diff
@@ -117,4 +122,4 @@
   <script>Backend.initScrollOffset()</script>
 
 </body>
-</html>
```
Comment (Toflar) [inline core-bundle/src/Resources/contao/templates/backend/be_main.html5]: You should configure your IDE correctly, this happens in all of your PRs somehow :D

File: .editorconfig [hunk 3]
```diff
@@ -13,7 +13,7 @@ trim_trailing_whitespace = true
 indent_style = space
 indent_size = 4
 
-[*.{svg,min.css,min.js}]
+[*.{svg,min.css,min.js,html5}]
```
Comment (ausi) [inline .editorconfig]: Unrelated ☺️

**Review by ausi — APPROVED** [review]

File: core-bundle/src/Resources/public/limit-height.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+        div.style.height = hgt+'px';
+
+        button.addEventListener('click', function () {
+            if (div.offsetHeight > hgt) {
+                div.style.height = hgt+'px';
+            } else {
+                div.style.height = 'auto';
+            }
+        });
```
Comment (leofeyer) [inline core-bundle/src/Resources/public/limit-height.js:43]: Do we no longer need the `button.set('data-state', button.get('data-state') ? 0 : 1);` line here?

Comment (aschempp) [reply core-bundle/src/Resources/public/limit-height.js:43]: I don't think we do, at least it has no technical benefit in this code. I can imagine we used it through CSS back in the days when we had that green down/up arrow to expand and collapse.

Comment (leofeyer) [reply core-bundle/src/Resources/public/limit-height.js:43]: Guess so…

**Review by leofeyer — APPROVED** [review]

**Pull request merged**